docs/source/examples/replicate_vicuna/translate_opts_py.yaml

```diff
@@ -1,10 +1,11 @@
-model_task: 'lm'
-model: finetuned_llama7B/llama7B-vicuna-onmt_step_4000.concat.pt
+model_task: "lm"
+model: finetuned_llama7B / llama7B - vicuna - onmt_step_4000.concat.pt
 transforms: [sentencepiece]
-src_subword_model: llama/tokenizer.model
-tgt_subword_model: llama/tokenizer.model
+src_subword_model: llama / tokenizer.model
+tgt_subword_model: llama / tokenizer.model
```

Comment:
Are these whitespaces expected?

Reply:
oh it seems that I applied black on the inference config file ...